maat-court-data-api/src/main/java/gov/uk/courtdata/reporder/service/RepOrderService.java

```diff
@@ -36,4 +36,11 @@ public void updateAppDateCompleted(final UpdateAppDateCompleted updateAppDateCom
         repOrderImpl.updateAppDateCompleted(updateAppDateCompleted.getRepId(), updateAppDateCompleted.getAssessmentDateCompleted());
         log.info("update app date completed  - Transaction Processing - End");
     }
+
+    @Transactional
+    public Boolean repOrderCountWithSentenceOrderDate(Integer repId) {
+        log.info("Get Rep order by appeal sentence order date is null - Transaction Processing - Start");
```

Comment:
Logging is misleading, should this be 'date is **NOT** null'?

Reply:
Thanks. Done now